src/corner_kick/src/store/actions/rosParameters.ts

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
I am not too familiar with JS, but is this file autogenerated?  What happens if we add more parameters in the yaml file, do we need to update these files or can we just call `catkin_make` again?

Reply:
This file is not auto-generated. Right now, we have it setting the 10 AI parameters from the Visualizer spec that we were given since we have the actions and reducers specifically written for those at the moment. Are there any more AI parameters that need to be added?

Reply:
Yea @akhilveeraghanta 	for now this is controlling the most common parameters that are the most useful to access from the main AI screen (since we don't want to be switching windows all the time to change stuff).

Also, @kelvinkoon 	do you want to wait for @akhilveeraghanta 's yaml list PR and update this branch, or handle that separately? It seems like the "mechanics" of updating parameters is in this PR so I'm totally fine doing it separately. Makes it easier to review too.

Reply:
@MathewMacDougall I'd prefer to handle that separately. I still have to make the rest of the parameters in the Visualizer spec, so it'd be easier to separate them.

Reply:
Sounds good